.circleci/config.yml

```diff
@@ -1,131 +1,114 @@
 version: 2
+aliases:
+  # From https://circleci.com/orbs/registry/orb/threetreeslight/puppeteer
+  - &install-puppeteer |
+    sudo apt-get install -yq \
+    gconf-service libasound2 libatk1.0-0 libatk-bridge2.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \
+    libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \
+    libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 \
+    libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates \
+    fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
 jobs:
-  build:
+  build_node_v10:
     docker:
       - image: circleci/node:10
```

Comment:
Not sure if you tried the `-browsers` variants of the containers:
https://circleci.com/docs/2.0/circleci-images/#language-image-variants

Reply:
Good catch; I hadn't heard of those.